packages/tfjs-node-helpers/src/testing/metrics/precision.ts

```diff
@@ -1,14 +1,21 @@
-import { ConfusionMatrix } from '../confusion-matrix';
+import { metrics } from '@tensorflow/tfjs-node';
 import { Metric } from '../metric';
-import { MetricCalculator } from '../metric-calculator';
+import { MetricCalculator, MetricCalculatorData } from '../metric-calculator';
 
 export class PrecisionMetricCalculator extends MetricCalculator {
-  public calculate(trueValues: Float32Array, predictedValues: Float32Array): Metric {
-    const { tp, fp } = new ConfusionMatrix(trueValues, predictedValues);
+  public calculate({ trueValues, predictedValues }: MetricCalculatorData): Metric {
+    const trueY = trueValues.as1D();
+    const predY = predictedValues.as1D();
+    const valueTensor = metrics.precision(trueY, predY);
+    const value = valueTensor.dataSync()[0];
```

Comment:
@VladimirGlinskii 

```suggestion
    const [value] = valueTensor.dataSync();
```

Also, it would be great if we could use the async variant of this call.